app/models/frontend/event.rb

```diff
@@ -146,6 +154,10 @@ def relive
       conference.metadata['relive']&.first { |r| r['guid'] == guid }
     end
 
+    def timelens_present?
+        not timeline_filename.blank? and not thumbnails_filename.blank?
```

Comment:
Da `present?` das [Gegenteil von !blank?](https://apidock.com/rails/Object/present%3F) ist, würde ich die positive Form bevorzugen: 
`timeline_filename.present? and thumbnails_filename.present?`

Reply:
Good point! Fixed and force-updated the branch. Feel free to merge this PR!